libr/core/cmd_search_rop.c

```diff
@@ -7,6 +7,8 @@
 #include "r_list.h"
 #include "r_types_base.h"
 
+#define COMA_HITS 2
```

Comment:
fixed again :)
what does it mean: initially you search the string for the first flag assignment, this marks the beginning of the flag only part, but you need to go back 2 commas to include the value for the first flag as well